### PR TITLE
Ks/getting-started-edits 

### PR DESCRIPTION
**Pull request by kelsey-steven-ada**
Add $ for shell prompt on shell examples and escape angle brackets in Followup Exercise so "\<NAME\>" & "\<AGE\>" placeholders show.

### CODE REVIEW

**Comment by audreyandoy** [comment]
LGTM 👍

File: 01-intro-to-python/00-intro-and-getting-started.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ Congratulations, you just wrote your first Python file!
 
 ## Followup Exercise
 
-Modify the "hello.py" program above to read in both your name and age and print out:  "Hello <NAME> you are <AGE> years old!".
+Modify the "hello.py" program above to read in both your name and age and print out:  "Hello \<NAME\> you are \<AGE\> years old!".
```
Comment (anselrognlie) [inline 01-intro-to-python/00-intro-and-getting-started.md:122]: I'd suggest using code quotes here which is in keeping with the previous passage, and allows us to remove the quotes, which students can find confusing early on.

I'd also suggest changing line 126 to use a code block (the triple backticks) to help it look like output (as with other commands and output run in this lesson).

```suggestion
Modify the "hello.py" program above to read in both your name and age and print out:  `Hello <NAME> you are <AGE> years old!`.
```

File: 01-intro-to-python/00-intro-and-getting-started.md [hunk 2]
```diff
@@ -40,28 +40,28 @@ We can run use a Python repl to run and test our code, but if we want to write l
 Open up terminal and check out the screen. It is a text-based visual of our folder and file structure. Let's find out where we are:
 
 ```sh
-pwd
+$ pwd
```
Comment (anselrognlie) [inline 01-intro-to-python/00-intro-and-getting-started.md:43]: Consider adding a callout nearby with a reminder that the `$` character represents their terminal prompt and should not be typed.

**Review by anselrognlie — APPROVED** [review]
Looks good to me. Just a couple suggestions.

File: 01-intro-to-python/00-intro-and-getting-started.md (hunk 2 above)
Comment (kelsey-steven-ada) [reply 01-intro-to-python/00-intro-and-getting-started.md:43]: Oo, nice call out, I'll add that.

File: 01-intro-to-python/00-intro-and-getting-started.md [hunk 3]
```diff
@@ -40,28 +40,35 @@ We can run use a Python repl to run and test our code, but if we want to write l
...
 ```
+### !callout-info
+
+## What's the `$` mean?
```
Comment (anselrognlie) [inline 01-intro-to-python/00-intro-and-getting-started.md:47]: A principle used for a lot of the curriculum is that the header of a section/callout should deliver the info directly. Applying that here, we might title this callout 
```suggestion
## `$` Represents the Prompt
```
(I know, there's a callout just below that violates this... 😆 )

File: 01-intro-to-python/00-intro-and-getting-started.md [hunk 4]
```diff
@@ -40,28 +40,35 @@ We can run use a Python repl to run and test our code, but if we want to write l
...
+
+This $ represents the command line prompt, and should not be typed in. This is a common style choice in command line tutorials and code examples, so we should get comfortable seeing it!
```
Comment (anselrognlie) [inline 01-intro-to-python/00-intro-and-getting-started.md:49]: We don't have one currently, but I expect similar information will be added to the installfest text or programming workflow, so this shouldn't be the first time students are made aware of this. I think it's definitely worth a reminder, but this sounds more like a first encounter than a memory aid. Maybe use just the first sentence?
```suggestion
This `$` represents the command line prompt, and should not be typed in.
```

**Review by anselrognlie — APPROVED** [review]
Some thoughts on the callout.

**Pull request merged**